documents/fundAccount.md

```diff
@@ -18,9 +18,10 @@ body, err := client.FundAccount.Create(data, nil)
 
 | Name          | Type        | Description                                 |
 |---------------|-------------|---------------------------------------------|
-| customer_id*   | string      | The id of the customer to be fetched  |
+| customerId*   | string      | The id of the customer to be fetched  |
```

Comment:
Parameter **customerId** should be same as mentioned in abv code

Reply:
[updated](https://github.com/razorpay/razorpay-go/blob/fund_doc_correction/documents/fundAccount.md?plain=1#L21)